src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_SIZE = 100;
+    private Random randomFigure = new Random();
+    private Random randomParameter = new Random();
+    private int dieNumber;
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        dieNumber = randomFigure.nextInt(5) + 1;

```

Comment:
5 is a magic number - make it a constant

Reply:
[not fixed](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)